src/bindings/tsd.ts

```diff
@@ -185,6 +185,11 @@ export class TSDBuilder extends ExportsWalker {
     if (!this.esm) {
       sb.push("declare namespace __AdaptedExports {\n");
       ++this.indentLevel;
+    } else {
+      if (this.program.options.exportMemory) {
+        sb.push("/** Exported memory */\n");
+        sb.push("export declare const memory: WebAssembly.Memory;\n");
```

Comment:
Might be that the `declare` keyword has to be removed here so it works with raw bindings.